exercises/clock/clock.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Clock do
+  defstruct hour: 0, minute: 0
+
+  @doc """
+  Returns a string representation of a clock:
+
+      iex> Clock.new(8, 9) |> to_string
+      "08:09"
+  """
+  @spec new(integer, integer) :: Clock
```

Comment:
This spec is totally valid, but it fixes the return value to the atom `:"Elixir.Clock"`.